src/mmw/js/src/core/models.js

```diff
@@ -154,6 +154,7 @@ var LayerModel = Backbone.Model.extend({
         legendUnitsLabel: null,
         legendUnitBreaks: null,
         bigCZ: null,
+        bringToFront: false
```

Comment:
Added `bringToFront` option for layers in order to allow some layers to be promoted to the front of the map when added. This fixes the issue in the case where the project AOI was over the weather station marker and so the popup would not work.